rake_helpers/generate_final_csvs.rb

```diff
@@ -64,7 +64,22 @@ def tidy_facebook_link(page)
     end
   end
 
-  task :name_list => :term_tables do
+  task :top_identifiers => :term_tables do
+    top_identifiers = @json[:persons].map { |p| p[:identifiers].map { |i| i[:scheme] } }.flatten.
```

Comment:
Looks like this line is throwing an error when people don't have identifiers - https://rollbar.com/everypolitician/rebuilder/items/48/ so probably needs a `.to_a` in there.

```
NoMethodError: undefined method `map' for nil:NilClass
/tmp/d20160317-3-jhfuzb/rake_helpers/generate_final_csvs.rb:68:in `block (3 levels) in <top (required)>'
/tmp/d20160317-3-jhfuzb/rake_helpers/generate_final_csvs.rb:68:in `map'
/tmp/d20160317-3-jhfuzb/rake_helpers/generate_final_csvs.rb:68:in `block (2 levels) in <top (required)>'
Tasks: TOP => default => csvs => term_csvs:name_list => term_csvs:top_identifiers
(See full trace by running task with --trace)
```